modules/osdk-csv-manual-annotations.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-csv-manual-annotations_{context}"]
+= Operator metadata annotations
+
+Operator developers can manually define certain annotations in the metadata of a cluster service version (CSV) to enable features or highlight capabilities in user interfaces (UIs), such as OperatorHub.
+
+The following table lists Operator metadata annotations that can be manually defined using `metadata.annotations` fields.
+
+.Annotations
+[cols="5a,5a",options="header"]
+|===
+|Field |Description
+
+|`alm-examples`
+|Provide custom resource definition (CRD) templates with a minimum set of configuration. Compatible UIs pre-fill this template for users to further customize.
+
+|`operatorframework.io/initialization-resource`
+|Specify a single required custom resource that must be created at the time that the Operator is installed. Must include a template that contains a complete YAML definition.
+
+|`operatorframework.io/suggested-namespace`
+|Set a suggested namespace where the Operator should be deployed.
+
+|`operators.openshift.io/infrastructure-features`
+|Infrastructure features supported by the Operator. Users can view and filter by these features when discovering Operators through OperatorHub in the web console. Valid values:
+
+- `disconnected`: Operator supports running in disconnected mode. All related images required for mirroring are listed by the Operator.
+- `proxy`: Operator supports running on a cluster behind a proxy. Required environment variables are passed down to Operands.
+- `fipsmode`: Operator supports running in FIPS mode.
```

Comment:
@spadgett Can you help confirm that these are the correct values for the infrastructure features? Looking at https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/index.ts#L27. Especially want to make sure that the FIPS one is accurate, as it looks to be `fipsmode`, though it was `fips` in the original description of https://issues.redhat.com/browse/CONSOLE-2096.

Reply:
Specifically, valid for 4.5 and onward.

Reply:
@spadgett Since this has gotten QE & peer review, I'm going to go ahead and merge this for `master` and `enterprise-4.7` so it can get in for translation / docs freeze. But if you have any feedback on this please LMK. I'll manually cherry-pick to 4.5 - 4.6 branches.